CommunityCalls/2017-07-05/agenda.md

```diff
@@ -0,0 +1,28 @@
+# DSC Resource Kit & Configurations Community Call - 07/05/2017
+
+**12PM (PST) on Wednesday, July 5th**  
+[ICS file](https://github.com/PowerShell/DscResources/blob/master/CommunityCalls/2017-07-05/2017-07-05CommunityCall.zip)
+
+## How to Join
+
+#### Skype for Business
+
+[Join Skype Meeting](https://meet.lync.com/microsoft/zachal/C8MCGVQB)  
+Trouble Joining? [Try Skype Web App](https://meet.lync.com/microsoft/zachal/C8MCGVQB?sl=1)
+
+#### Phone
+
+Toll-free number: +1 (866) 641-7188 
+Toll number: +1 (773) 917-4061 
+Conference ID: 596307  (Dial-in Number)  
+[Find a local number](https://dialin.lync.com/8551f4c1-bea3-441a-8738-69aa517a91c5?id=68893190)  
```

Comment:
Is this URL updated? Not sure if it matters, but looks like it still has the access code from the last call.